### DISCUSSION TITLE
Adding tap-interfaces to sensor

### DISCUSSION THREAD

**Discussion by ejgh-oe** [Security-Onion-Solutions/securityonion#4316] [Unsupported Versions] [answered]
Hi,
Just installed SO 2.3.51 in a distributed environment, i.e. sensor, search node and manager (VM). Upon installation of the sensor node the installer came up with a question about the management interface and let me select only *one* capture interface.

However the box in question has several physical interfaces and I'd like the sensor to sniff traffic from all those interfaces (these interfaces connect to different LAN-switches that mirror traffic to the respecitive SPAN port).

So my question: how can I (re)configure the sensor so that it grabs traffic from additional interfaces? (So this is not about bonding interfaces together, but to really have different interfaces)
Unfortunately I couldn't find an answer in the docs which is the reason I'm posting my question here ;-)

Thanks much in advance for any clue,
-ej

**Comment by Shak3** [comment]
is the sensor a VM also?

**Comment by ejgh-oe** [comment]
Nope, sensor is a hardware box (HP server) with 6 GE interfaces

**Comment by gcsrellis** [comment]
All of the interfaces did show, but only 1 could be selected correct?

**Reply by ejgh-oe** [reply]
> 
> 
> All of the interfaces did show, but only 1 could be selected correct?

Yes, exactly

**Comment by rwwiv** [comment, marked as answer]
I think you may be misunderstanding the purpose of the management interface. A sensor isn't actually sniffing traffic on that port, it's using it to communicate with the manager. You should have seen another menu with the instructions `Please add NICs to the Monitor Interface`, which was where you were able to select multiple interfaces on which to sniff traffic.

As far as your question, there is a script called `so-monitor-add` which you can use to add interfaces to the bond set up for traffic sniffing/monitoring (`bond0`). Bonding the interfaces together like this is how Security Onion monitors traffic across each port. 

However, you may want to verify you're seeing traffic coming from that sensor first since it sounds like it may have been mis-configured during setup.

**Reply by ejgh-oe** [reply]
Hi,
> I think you may be misunderstanding the purpose of the management interface. A sensor isn't actually sniffing traffic on that port, it's using it to communicate with the manager. You should have seen another menu with the instructions `Please add NICs to the Monitor Interface`, which was where you were able to select multiple interfaces on which to sniff traffic.

Yes, during setup after selecting the NIC for the management interface I was given the option to add one interface as a monitoring interface, but I need several monitoring interfaces.

> 
> As far as your question, there is a script called `so-monitor-add` which you can use to add interfaces to the bond set up for traffic sniffing/monitoring (`bond0`). Bonding the interfaces together like this is how Security Onion monitors traffic across each port.

That is exactly what I was looking for :-) - so all monitoring interfaces are bound together in a "bond" interface, that's the point.

> 
> However, you may want to verify you're seeing traffic coming from that sensor first since it sounds like it may have been mis-configured during setup.

Yes, SO is fully functional so far with one management interface and one monitoring interface - my question was only about enabling additional monitoring interfaces.

**Reply by rwwiv** [reply]
> Yes, during setup after selecting the NIC for the management interface I was given the option to add one interface as a monitoring interface, but I need several monitoring interfaces.

Sounds like we may need to be more clear about how to select multiple choices in that menu then, it does in fact support choosing more than one interface.

> That is exactly what I was looking for :-)

Glad to hear I was able to help, I'll mark this as the answer then.
